crates/test-utils/src/messages.rs

```diff
@@ -183,7 +180,9 @@ pub fn make_transactions_with_pre_genesis_objects(
             /* gas_object_ref */ o2.compute_object_reference(),
             MAX_GAS,
         );
-        let signature = Signature::new(&data, &signer);
+        let signature = keys
+            .sign(&o1.owner.get_owner_address().unwrap(), &data.to_bytes())
```

Comment:
I assume that at this stage unwrap() will always succeed. Can we add a comment, cause eventually we need to clean up all unwraps to avoid halting if an error occurs (unless we know unwrap() is safe)